app/src/services/storage.js

```diff
@@ -86,6 +92,34 @@ const objectStorageService = {
     return this._s3Client.send(new DeleteObjectCommand(params));
   },
 
+  /**
+   * @function deleteObjectTagging
+   * Deletes the tags of the object at `filePath`
+   * @param {string} options.filePath The filePath of the object
+   * @param {number} [options.versionId] Optional specific versionId for the object
+   */
+  async deleteObjectTagging({ filePath, tags, versionId }) {
+    const params = {
+      Bucket: bucket,
+      Key: filePath,
+    };
+
+    if (versionId) params.VersionId = versionId;
+
+    return this._s3Client.send(new DeleteObjectTaggingCommand(params));
+  },
+
+  /**
+   * @function getBucketEncryption
+   */
+  async getBucketEncryption() {
+    const params = {
+      Bucket: bucket
+    };
+
+    return await this._s3Client.send(new GetBucketEncryptionCommand(params));
```

Comment:
We should not be awaiting and force unwrapping this command as the storage service by convention always returns a promise.